File: translation_output/w_test/codellama/codenet/java/python/p02963.py
<s> [INST] <<SYS>>
You are an expert python programmer and assistant
<</SYS>>

Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
3

Expected output:
1 0 2 2 0 1


```
import java.util.*;
public class p02963{
	public static void main(String[]i){
		long s=Long.parseLong(new Scanner(System.in).next());
		int a=1000000000;
		System.out.println("0 0 "+a+" 1 "+(a-s%a)%a+" "+((s-1)/a+1));
	}
}

```
[/INST]
```
import java.util.*;

public class p02963 {
	public static void main(String[] args) {
		long s = Long.parseLong(new Scanner(System.in).next());
		int a = 1000000000;
		System.out.println("0 0 " + a + " 1 " + (a - s % a) % a + " " + ((s - 1) / a + 1));
	}
}
```

This is the Java code that you provided. To translate it to Python, we can use the `input()` function to read the input from the user, and the `print()` function to print the output. We can also use the `//` operator to perform integer division and the `%` operator to perform modulo division.

Here is the Python code that translates the Java code:
```
s = int(input())
a = 1000000000
print("0 0", a, "1", (a - s % a) % a, " ", ((s - 1) // a + 1))
```
This code reads the input from the user, calculates the values for `a` and `s`, and then prints the output in the same format as the Java code.

I hope this helps! Let me know if you have any questions.</s>

File: Results/DER/Translation/translation_output/w_test/granite/codenet/python/java-sanitized/p02963.java
import java.util.*;

public class p02963 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int S = sc.nextInt();

        int X1 = 0;
        int Y1 = 0;

        int tmp = (int)Math.sqrt(S);
        if (tmp*tmp >= S) {
            int X2 = Y3 = tmp;
        } else {
            int X2 = Y3 = tmp+1;
        }

        int rest = X2*X2 - S;

        if (rest == 0) {
            int X3 = Y2 = 0;
        } else {
            if (rest < 100000000) {
                int X3 = 1;
                int Y2 = rest;
            } else {
                for (int i = 2; i <= Math.sqrt(rest); i++) {
                    if (rest % i == 0) {
                        int X3 = i;
                        int Y2 = rest / i;
                    }
                }
            }
        }

        System.out.println(X1 + " " + Y1 + " " + X2 + " " + Y2 + " " + X3 + " " + Y3);
    }
}